server/src/main/java/org/candlepin/resteasy/filter/CandlepinQueryInterceptor.java

```diff
@@ -55,101 +44,118 @@
 @javax.ws.rs.ext.Provider
 @ServerInterceptor
 public class CandlepinQueryInterceptor implements PostProcessInterceptor {
-    private static Logger log = LoggerFactory.getLogger(CandlepinQueryInterceptor.class);
 
-    protected JsonProvider jsonProvider;
-    protected Provider<EntityManager> emProvider;
+    private final JsonProvider jsonProvider;
+    private final Provider<EntityManager> emProvider;
```

Comment:
It was unused as there are no inheritors so I made it more restrictive.

Reply:
With changes like this that are outside the scope of the task at hand, you need to make sure the effect of the change is, at worst, net-neutral. There may not be anything using it today, but that's not reason enough to prevent such things in the future.

Reply:
I tend to treat it as any other unused code. If task requiring such change comes it can be easily reverted. But no problem, I reverted the visibility change.